core/common/src/main/java/alluxio/PropertyKey.java

```diff
@@ -2520,10 +2520,11 @@ public String toString() {
           .build();
   public static final PropertyKey USER_NETWORK_NETTY_CHANNEL =
       new Builder(Name.USER_NETWORK_NETTY_CHANNEL)
-          .setDescription("Type of netty channels.")
+          .setDescription("Type of netty channels. If EPOLL is not available, this will "
+              + "automatically fall back to NIO.")
           .setConsistencyCheckLevel(ConsistencyCheckLevel.WARN)
           .setScope(Scope.CLIENT)
-          .setDefaultSupplier(ChannelType.DEFAULT_SUPPLIER)
```

Comment:
shall we remove `ChannelType .DEFAULT_SUPPLIER` since it is no more used?

Reply:
done